source/index.js

```diff
@@ -1,2 +1,7 @@
-require( "./angular-date-picker" );
-require( "./angular-time-picker" );
+"use strict";
+
+angular.module( "vokal.datePicker", [] )
+.directive( "datePicker", require( "./directives/date-picker" ) );
+
+angular.module( "vokal.timePicker", [] )
+.directive( "timePicker", require( "./directives/time-picker" ) );
```

Comment:
I was going to say something but figured someone could just load them in their code like you are here if they only wanted one of them. If you want to make it friendlier than that I don't think it would hurt.